examples/tutorials/starting/analysis_2.py

```diff
@@ -297,7 +297,14 @@
 # the ``stats`` method of the dataset to the minimizer. By default, it
 # uses ``iminuit``.
 #
-# Its constructor takes a list of dataset as argument.
+# It is possible that the fit will try to test positions far from your source of interest and fail to converge
+# therefore you may choose to constrain the model parameter ranges
+# to stay within your expected region (see :ref:`here <dropdown-improve-fit>`).
+# For further information on fitting you can see the :doc:`/tutorials/details/fitting`
+# overview tutorial or :ref:`this section <modifying-model-parameters>`
+# about modifying your parameters.
+#
+# The fit constructor takes a list of datasets as argument.
```

Comment:
Well this is not true since v0.20!!! We should have changed that a while ago...

Reply:
write instead that it takes dictionaries defining global options for the optimizer